src/TritonRoute/src/gr/FlexGR_topo.cpp

```diff
@@ -51,8 +52,9 @@ void FlexGR::genSTTopology_FLUTE(vector<frNode*>& pinGCellNodes,
     xs[i] = loc.x();
     ys[i] = loc.y();
   }
-  int accuracy = 3;
-  auto fluteTree = stt::flute(degree, xs.data(), ys.data(), accuracy);
+  // temporary to keep using flute here
+  stt_builder_->setAlpha(0);
+  auto fluteTree = stt_builder_->buildSteinerTree(xs, ys, 0);
```

Comment:
this assumes pin 0 is the driver, which appears to only be a guess

Reply:
Currently, TR will only use flute. After the TR update (which will add dbNet information) I will add a function to correctly detect the driver.